### ISSUE TITLE
Setting tabs.focus_stack_size to -1 causes crash on startup.

### ISSUE THREAD

**Issue by cadadr** [qutebrowser/qutebrowser#5742]
**Version info**:

qutebrowser v1.10.1
Git commit: 
Backend: QtWebEngine (Chromium 69.0.3497.128)
Qt: 5.12.8

CPython: 3.8.2
PyQt: 5.14.1

Linuxmint 20 package, details:

https://paste.the-compiler.org/view/c288cead

**Does the bug happen if you start with `--temp-basedir`?**:

Yes, with the following invocation:

    qutebrowser -T -s tabs.focus_stack_size -1

**Description**

The help for `tabs.focus_stack_size` reads:

    Maximum stack size to remember for tab switches (-1 for no maximum).

When I set this value to -1, however, this causes an unhandled exception during startup:

```
==== Exception ====
Traceback (most recent call last):
  File "/usr/bin/qutebrowser", line 11, in <module>
    load_entry_point('qutebrowser==1.10.1', 'gui_scripts', 'qutebrowser')()
  File "/usr/lib/python3/dist-packages/qutebrowser/qutebrowser.py", line 201, in main
    return app.run(args)
  File "/usr/lib/python3/dist-packages/qutebrowser/app.py", line 123, in run
    init(args=args)
  File "/usr/lib/python3/dist-packages/qutebrowser/app.py", line 163, in init
    _process_args(args)
  File "/usr/lib/python3/dist-packages/qutebrowser/app.py", line 203, in _process_args
    window = mainwindow.MainWindow(private=None)
  File "/usr/lib/python3/dist-packages/qutebrowser/mainwindow/mainwindow.py", line 230, in __init__
    self.tabbed_browser = tabbedbrowser.TabbedBrowser(win_id=self.win_id,
  File "/usr/lib/python3/dist-packages/qutebrowser/mainwindow/tabbedbrowser.py", line 237, in __init__
    self.tab_deque = TabDeque()
  File "/usr/lib/python3/dist-packages/qutebrowser/mainwindow/tabbedbrowser.py", line 67, in __init__
    self._stack = collections.deque(
ValueError: maxlen must be non-negative
```

**How to reproduce**

The following command should suffice:

    qutebrowser -T -s tabs.focus_stack_size -1

**Comment by The-Compiler** [comment]
This was already fixed in v1.13.0 via ae19c133e1dc790905aafa52e92bca235d2a2ce31 but you're running an older version.

**Comment by cadadr** [comment]
FWIW, this also caused me to lose the active session file, tho it was backed up in adjacent `default.yml`.
